ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable.yaml

```diff
@@ -259,6 +259,16 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: rosa-aws-non-sts-advanced
+- as: aws-ipi-f7-netobserv
```

Comment:
4.12 add into stable.yaml, while 4.13 and above add into nightly.yaml.   Is it expected ?